README.md

```diff
@@ -37,6 +37,7 @@ Options are configured via environment variables.
 | LOGGER_LEVEL               | No       | info                | Severity level from which to start showing errors eg info will hide debug messages                                           |
 | LOGGER_FORMAT              | No       | simple              | Simple or JSON log output: simple/json                                                                                       |
 | LOGGER_SECRETS             | No       | ["secret","secret"] | String array of exact secrets to sanitize in logs                                                                            |
+| VALIDATOR_EXIT_WEBHOOK     | No       | http://webhook      | POST validator info to an endpoint instead of sending out an exit message when exit is initiated                             |
```

Comment:
hey guys, we're looking forward to try this PR once it's merged. we're wondering if `MESSAGES_LOCATION` should still be required when `VALIDATOR_EXIT_WEBHOOK` is set. it seems that `validator-ejector` should only allow one of these variables to be defined.

to ilustrate what we were expecting, say `MESSAGES_LOCATION` in [src/service.ts](https://github.com/lidofinance/validator-ejector/blob/44f069aa33b3b19758a55a150a84d2d41be07002/src/services/config/service.ts) is marked as optional.

then, in the beginning of `bootstrap` in [app/module.ts](https://github.com/lidofinance/validator-ejector/blob/74339016d5fb107d5004c958e3e651e8419ee244/src/app/module.ts) you guys could make sure only one of these variables are set:

```ts
if(MESSAGES_LOCATION && VALIDATOR_EXIT_WEBHOOK) {
	logger.error('MESSAGES_LOCATION and VALIDATOR_EXIT_WEBHOOK are defined. Ensure only one env variable is defined', e)
	process.exit(1)
}

if(!MESSAGES_LOCATION && !VALIDATOR_EXIT_WEBHOOK) {
	logger.error('MESSAGES_LOCATION or VALIDATOR_EXIT_WEBHOOK are not defined. Ensure only one env variable is defined', e)
	process.exit(1)
}
```

and finally ensure `messagesProcessor.load()` and `messagesProcessor.verify()`  in [app/service.ts](https://github.com/lidofinance/validator-ejector/blob/44f069aa33b3b19758a55a150a84d2d41be07002/src/app/service.ts) don't run if `MESSAGES_LOCATION` is not set.

note 1: we're not familiar with the codebase so likely there's a better approach.
note 2: looking to the PR, it should work fine if `MESSAGES_LOCATION` points to an empty folder. but the configuration and usage will look a bit weird for users looking to use `VALIDATOR_EXIT_WEBHOOK`

Reply:
Hey @joaocenoura, thanks for the comment! You are right, we'll try to do just that.

Reply:
@joaocenoura Done!